Webhooks slack guide expansion (#2957)
fishtown-analytics/docs.getdbt.com: website/docs/guides/orchestration/webhooks/zapier-refresh-mode-report.md

Zapier allows you to [store secrets](https://help.zapier.com/hc/en-us/articles/8496293271053-Save-and-retrieve-data-from-Zaps), which prevents your keys from being displayed in plaintext in the Zap code. You will be able to access them via the [StoreClient utility](https://help.zapier.com/hc/en-us/articles/8496293969549-Store-data-from-code-steps-with-StoreClient).

> This guide assumes secrets are stored under the key names `DBT_WEBHOOK_KEY`, `MODE_API_TOKEN` and `MODE_API_SECRET`. If you choose different names, make sure you update all references to them in the sample code.

Reviewer: ```suggestion
This guide assumes the names for the secret keys are: `DBT_WEBHOOK_KEY`, `MODE_API_TOKEN`, and `MODE_API_SECRET`. If you are using different names, make sure you update all references to them in the sample code.
```

> In this guide we will use a short-lived code action to store the secrets, but you can also use the [REST API via Postman](https://store.zapier.com/) or create a separate Zap and call the [Set Value Action](https://help.zapier.com/hc/en-us/articles/8496293271053-Save-and-retrieve-data-from-Zaps#3-set-a-value-in-your-store-0-3).

Reviewer: ```suggestion
This guide uses a short-lived code action to store the secrets, but you can also use the [REST API from Postman](https://store.zapier.com/), or create a separate Zap and call the [Set Value Action](https://help.zapier.com/hc/en-us/articles/8496293271053-Save-and-retrieve-data-from-Zaps#3-set-a-value-in-your-store-0-3).
```

Author: Does "from" here imply that Postman created the API? It's still a Zapier api, it's just that we're using Postman to interact with it

Reviewer: great clarifying question! your extra context is helpful here! so it's more like _by way of_ Postman

suggest "REST API through Postman" or  "REST API with Postman"

Author: I might do "use a tool like Postman to interact with the REST API"

Reviewer: yes! i like that!

#### a. Create a Storage by Zapier connection

> If you haven't already got one, go to <https://zapier.com/app/connections/storage> and create a new connection, making note of the UUID secret you generate.

Reviewer: ```suggestion
If you haven't already got one, go to <https://zapier.com/app/connections/storage> and create a new connection. Remember the UUID secret you generate for later. 
```

#### b. Add a temporary code step
Choose **Run Python** as the Event. Run the following code: 
```python 
store = StoreClient('abc123') #replace with your UUID secret
store.set('DBT_WEBHOOK_KEY', 'abc123') #replace with your dbt Cloud API token
store.set('MODE_API_TOKEN', 'abc123') #replace with your Mode API Token
store.set('MODE_API_SECRET', 'abc123') #replace with your Mode API Secret
```

> Test the step. Once it has succeeded, you can delete this Action. The key will remain stored as long as it is accessed at least once every 3 months.

Reviewer: ```suggestion
Test the step. You can delete this Action when the test succeeds. The key will remain stored as long as it is accessed at least once every three months.
```
